ui/src/views/mint/github-step/steps/github-connect/github-connect-step.tsx

```diff
@@ -7,13 +7,18 @@ export const GithubConnect: React.FC = () => {
   const { prevStep } = Stepper.useContext();
 
   return (
-    <Card.Container>
+    <Card.Container css={{ maxWidth: '$107h' }}>

```

Comment:
Seems like all `Card.Container` usages uses this same `maxWidth`, should it be a property from the component?

Reply:
Yes, I was thinking to work on that refactor today (Card Header and this Card component that is for mint and access point flows)